common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -84,6 +83,15 @@ private static String toPoolName(Class<?> poolType) {
         }
     }
 
+    public DefaultThreadFactory(String poolName, boolean daemon, int priority) {
+        this(
```

Comment:
nit: formatting seems strange here. can we get rid of the extra line and spaces?